conductr_cli/host.py

```diff
@@ -5,16 +5,20 @@
 from conductr_cli.docker import DockerVmType
 
 
+CONDUCTR_HOST = 'CONDUCTR_HOST'
+CONDUCTR_IP = 'CONDUCTR_HOST'
```

Comment:
Yes - it should. Thank you for picking this up.

I need to add the test for `host.py`